### DISCUSSION TITLE
Collapse links not working?

### DISCUSSION THREAD

**Discussion by NinjaArmy** [saadeghi/daisyui#1434] [Q&A] [answered]
I'm using multiple collapse items with focus: https://daisyui.com/components/collapse/ from the docs.
I need the focus behavior because I only want to have one item open at a time.

I think there is a bug and couldn't find an easy solution. See this code example 
```html
<div tabindex="0" class="collapse"> 
  <div class="collapse-title text-xl font-medium">
    Focus me to see content
  </div>
  <div class="collapse-content"> 
    <p>tabindex="0" attribute is necessary to make the div focusable</p>
    <a class="btn" href="">Link</a>
  </div>
</div>
```
The button is unreachable. If you try to click the button the collapse item just closes itself because it loses focus. Is there an easy workaround? 

I'm using nuxt3. And the collapse items come from a backend so working with the open and close classes also seemed to be difficult. Somone has an idea?

**Comment by saadeghi** [comment, marked as answer]
There are 2 types of `collapse` in the docs: https://daisyui.com/components/collapse/  
One works with `focus` and the other works with a checkbox.  

The one that works focus, only gets open when the div is focused, when div loses focus, it gets closed. So when any other element (for example a link inside it) gets a focus, it closes instantly.  

This is a limitation when it works with CSS `focus`. It works fine for text or non-interactive content but if you have links, etc you may want to use the collapse with checkbox.

Let me know if you have any questions.

**Reply by kimdre** [reply]
Is it possible to close the opened collapse-content on another click in the container just like you open it or is this only possible with the checkbox-collapse example?
I would like to add the collapse-arrow class but this doesn't seem to work with the checkbox-collapse.

Edit: Ok, I figgured it out. This was actually easier than I thought.
Just add this to your custom .css file
```css
.collapse:focus {
    @apply pointer-events-none;
}
```

**Comment by NinjaArmy** [comment]
Thanks for the answer!

I saw the option with the checkbox as well. 
Unfortunately I can't create the intended behavior with the checkbox since I always want to have only one item open.
